components/x-audio/src/redux/index.jsx

```diff
@@ -65,6 +75,23 @@ export default function connectPlayer (Player) {
 			if (this.playingStateAndPropsNeedSync()) {
 				this.updatePlayingStateFromProps(prevProps);
 			}
+
+			if (this.expandedPlayerRef && !this.hasSetSwipeAnimation) {
+				this.setSwipeDownAnimation()
+			}
+		}
+
+		setSwipeDownAnimation () {
+			const hammer = new Hammer.Manager(this.expandedPlayerRef);
+			hammer.add(new Hammer.Pan({
+				direction: Hammer.DIRECTION_ALL,
```

Comment:
could this be `DIRECTION_DOWN` or `DIRECTION_VERTICAL` instead?

Reply:
👍